experiments/chaos/kubernetes/pod_delete/run_litmus_test.yml

```diff
@@ -3,18 +3,25 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   generateName: pod-delete-
-  namespace: litmus
+
+  # Executed on application's namespace in case of automated runs 
+  # Provide appropriate namespace if executed manually 
+  # namespace: litmus
 spec:
   template:
     metadata:
       labels:
         experiment: pod-delete
     spec:
-      serviceAccountName: litmus
+      # Placeholder that is updated by the executor for automated runs
+      # Provide appropriate SA (with desired permissions) if executed manually
+      serviceAccountName: CHAOS_SERVICE_ACCOUNT
+
       restartPolicy: Never
       containers:
       - name: ansibletest
-        image: openebs/ansible-runner:ci
+        #image: openebs/ansible-runner:ci
+        image: ksatchit/ansible-runner:trial7
```

Comment:
@ksatchit can you change the runner image into openebs/ansible-runner:ci

Reply:
done